.envrc

```diff
@@ -0,0 +1,3 @@
+source_url "https://raw.githubusercontent.com/cachix/devenv/d1f7b48e35e6dee421cfd0f51481d17f77586997/direnvrc" "sha256-YBzqskFZxmNb3kYVoKD9ZixoPXJh1C9ZvTLGFRkauZ0="
```

Comment:
is it necessary to pin(with revision and sha hash) the script here ?

Reply:
Thanks for the review :) Yup, this is used by direnv.